sdk/storage/Azure.Storage.Files.DataLake/src/Azure.Storage.Files.DataLake.csproj

```diff
@@ -20,5 +20,7 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Storage.Common\src\Azure.Storage.Common.csproj" />
+    <ProjectReference Include="..\..\Azure.Storage.Blobs\src\Azure.Storage.Blobs.csproj" />
+    <ProjectReference Include="..\..\Azure.Storage.Files\src\Azure.Storage.Files.csproj" />
```

Comment:
This references Files too?

Reply:
It doesn't, but oddly when I remove this project reference, I'm no longer able to reference System.Text.Json.

Reply:
fixed.